src/atoms/Typography.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+import { Heading, SubHeading, Body, Code } from './Typography';
+
+<Meta title="atoms/Typography" parameters={{ actions: { argTypesRegex: '^on.*' } }} />
+
+# Typography
+
+Typography components
+
+<Canvas>
+  <Story name="Heading">
+    <Heading>Heading</Heading>
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="SubHeading">
+    <SubHeading>SubHeading</SubHeading>
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="Body">
+    <Body>Body</Body>
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="Code">
+    <Code>Code</Code>
+  </Story>
+</Canvas>
```

Comment:
Can you make the text editable here?
https://storybook.js.org/docs/react/essentials/controls